### ISSUE TITLE
"Phone" icon is taking the "contacts" icon on Samsung phones.

### ISSUE THREAD

**Issue by vordx** [8bitPit/Niagara-Issues#315]
Using note 8 , Oreo , 0.4 build 4 of the launcher.

https://vgy.me/m9dFP4.jpg , interesting this bug was reported before on ruthless launcher.

**Comment by M4rku5in** [comment]
#metoo

**Comment by 8bitPit** [comment]
@vordx & @M4rku5in have you also tested it on other Launchers e.g. Lawnchair or Nova?

**Comment by M4rku5in** [comment]
Yes, I tested. I used a launcher Nova and there it was the same. But I did not do it because Nova allowed the icons to change.

**Comment by 8bitPit** [comment]
@M4rku5in then it seems that Samsung did something wrong

**Comment by M4rku5in** [comment]
I using LG G6 phone, android 8.0. I am waiting for change icon...

**Comment by vordx** [comment]
@8bitpit , edited my post

**Comment by 8bitPit** [comment]
@vordx so you're using a custom icon pack?

**Comment by vordx** [comment]
@8bitpit , yes but it's working fine with other launchers ,   just reporting the bug nothing more

**Comment by 8bitPit** [comment]
It should be fixed by now (v0.6.0), can you confirm it?

**Comment by M4rku5in** [comment]
Fixed. Everything works correctly.
